v0/pipeline/steps/cd/k8s-traffic-routing-provider-base-spec.yaml

```diff
@@ -2,8 +2,6 @@ title: K8sTrafficRoutingProviderBaseSpec
 type: object
 allOf:
   - "$ref": "k8s-traffic-routing-destinations.yaml"
-required:
```

Comment:
I am not getting this change. We should have routes with at least 1 item in array.

Reply:
for SMI provider it's not required